scripts/qa_crowbarsetup.sh

```diff
@@ -3273,14 +3273,16 @@ function onadmin_run_cct()
     # - install cct dependencies
     ensure_packages_installed git-core gcc make ruby2.1-devel
 
-    # - default checkout location for mkcloud based setups:
-    mkdir -p /root/github.com/SUSE-Cloud
-    pushd /root/github.com/SUSE-Cloud/
-    git clone https://github.com/SUSE-Cloud/cct.git
-    cd cct
-    bundle install
-
-    if [[ -n $cct_tests ]]; then
+    if iscloudver 6plus && [[ -n $cct_tests ]]; then
+
+        # - default checkout location for mkcloud based setups:
+        mkdir -p /root/github.com/SUSE-Cloud
+        pushd /root/github.com/SUSE-Cloud/
+        git clone https://github.com/SUSE-Cloud/cct.git
+        # TODO checkout branches for older versions
+        cd cct
+        bundle install
```

Comment:
Good question but this code is already merged via #446.  BTW see https://github.com/bundler/bundler/issues/3006 which just hit me (again) in a completely different way :(